src/couch/src/couch_passwords.erl

```diff
@@ -37,18 +37,18 @@ hash_admin_password(ClearPassword) when is_list(ClearPassword) ->
     hash_admin_password(?l2b(ClearPassword));
 hash_admin_password(ClearPassword) when is_binary(ClearPassword) ->
     %% Support both schemes to smooth migration from legacy scheme
-    Scheme = config:get("couch_httpd_auth", "password_scheme", "pbkdf2"),
+    Scheme = chttpd_util:get_chttpd_auth_config("password_scheme", "pbkdf2"),
     hash_admin_password(Scheme, ClearPassword).
 
 hash_admin_password("simple", ClearPassword) -> % deprecated
     Salt = couch_uuids:random(),
     Hash = crypto:hash(sha, <<ClearPassword/binary, Salt/binary>>),
     ?l2b("-hashed-" ++ couch_util:to_hex(Hash) ++ "," ++ ?b2l(Salt));
 hash_admin_password("pbkdf2", ClearPassword) ->
-    Iterations = config:get("couch_httpd_auth", "iterations", "10000"),
+    Iterations = chttpd_util:get_chttpd_auth_config("iterations", "10"),
```

Comment:
Please change the default value back to "10000"

Reply:
On the https://github.com/apache/couchdb/pull/3577/files#diff-47040d0b65ca2cdd2c1a8eea22187c860de02e51bd67ebf14c8287f5b56b26e8R82 the `chttpd_util:get_chttpd_auth_config_integer` is used. Is it intentional to not use it here?